Emulation Page Update (#286)
rancher-sandbox/docs.rancherdesktop.io: docs/ui/preferences/virtual-machine/emulation.md

  <link rel="canonical" href="https://docs.rancherdesktop.io/ui/preferences/virtual-machine/emulation"/>
</head>

### QEMU

 ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation.png)

> - The "[QEMU](https://www.qemu.org/documentation/)" option is enabled by default and is used to run a guest operating system.

Reviewer: I'm not familiar with how other pages in this set are formatted, but seeing bullet points used for captions feels a little odd, since they're single items and  not part of a list. If the bullets are intended to make the text line up under the image, could you indent instead?

Author: Tbh not sure why I bulleted and agree with your points, removing the formatting.

> - The "[QEMU](https://www.qemu.org/documentation/)" option is enabled by default and is used to run a guest operating system.

Reviewer: ```suggestion
- The [**QEMU**](https://www.qemu.org/documentation/) option enables or disables a guest operating system. It is enabled by default.
```

### VZ

 ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation_vz.png)

:::caution warning

> This is an **experimental** setting. The setting is available for ARM machines on  macOS 13.3 or later, and Intel machines on macOS 13.0 or later.

Reviewer: ```suggestion
This is an *experimental* setting. The setting is available for ARM machines on  macOS 13.3 or later, and Intel machines on macOS 13.0 or later.
```

We should generally reserve bold for UI label syntax, for consistency

:::

> - The "VZ" option can also be enabled to make use of the native macOS [Virtualization.Framework](https://developer.apple.com/documentation/virtualization) for running a guest machine.

Reviewer: ```suggestion
- You can also enable the **VZ** option to use the native macOS [Virtualization.Framework](https://developer.apple.com/documentation/virtualization) for running a guest machine.
```

>   - **VZ option**:

Reviewer: Nitpick: the bold should go around the text of the label (**VZ** option, not **VZ option**)

 ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation.png)

> The [**QEMU**](https://www.qemu.org/documentation/) option is enabled by default and is used to run a guest operating system. You can switch the virtual machine type after the first run.

Reviewer: ```suggestion
The [**QEMU**](https://www.qemu.org/documentation/) option is enabled by default and turns on a guest operating system. You can switch the virtual machine type after the first run.
```

I guess I'm quibbling over the option being used to run something (like electricity to a device) vs turning a feature on (like flipping a power switch)

Author: Just applied!

### VZ

 ![](https://suse-rancher-media.s3.amazonaws.com/desktop/v1.10/preferences/macOS_virtualMachine_tabEmulation_vz.png)

:::caution warning

This is an *experimental* setting. The setting is available for ARM machines on  macOS 13.3 or later, and Intel machines on macOS 13.0 or later.

:::

You can also enable the **VZ** option to use the native macOS [Virtualization.Framework](https://developer.apple.com/documentation/virtualization) for running a guest machine.

> The suboption **VZ option** can also be enabled if using **VZ** as your virtual machine type. The suboption allows for you to enable [Rosetta support](https://developer.apple.com/documentation/virtualization/running_intel_binaries_in_linux_vms_with_rosetta) and makes it possible for users to run applications that contain x86_64 instructions on Apple hardware.

Reviewer: ```suggestion
The suboption **VZ option** can also be enabled if using **VZ** as your virtual machine type. The suboption enables [Rosetta support](https://developer.apple.com/documentation/virtualization/running_intel_binaries_in_linux_vms_with_rosetta) and allows users to run applications that contain x86_64 instructions on Apple hardware.
```

Author: Just applied!
